.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+*.swo
+*.swp
```

Comment:
Just a note here, not a big deal, but these two are usually ignored in your home `gitignore` because are specific for your IDE 😄